ltm.go

```diff
@@ -960,14 +967,6 @@ func (b *BigIP) Monitors() ([]Monitor, error) {
 // CreateMonitor adds a new monitor to the BIG-IP system. <parent> must be one of "http", "https",
 // "icmp", or "gateway icmp".
 func (b *BigIP) CreateMonitor(name, parent string, interval, timeout int, send, receive string) error {
-	if strings.Contains(send, "\r\n") {
-		send = strings.Replace(send, "\r\n", "\\r\\n", -1)
-	}
-
-	if strings.Contains(parent, "gateway") {
```

Comment:
@scottdware - Any reason this was here instead of throwing an error for invalid parent type?